test/integration/logging/local_live_log_test.js

```diff
@@ -78,6 +76,23 @@ suite('live logging', () => {
       token.length,
       22, 'Token should be 22 characters long'
     );
+  });
 
+  test('live log expiration date', async () => {
+    let expiration = taskcluster.fromNowJSON('1 second');
```

Comment:
Hrm, is this sufficiently enough time for the task to be submitted and claimed consistently? anything less than one year would work since the default is now() + 1 year for the default expiration.